### ISSUE TITLE
Issue loading the .riv file when configured using webpack.

### ISSUE THREAD

**Issue by vigsub** [rive-app/rive-wasm#298]
I'm using webpack 5 for bundling and trying to load a rive file into my app. I'm getting the following error.

![Screenshot 2023-03-18 at 12 03 46 AM](https://user-images.githubusercontent.com/21095278/225989606-29b3054d-1c61-439f-b162-ad53ed6d8b29.png)

As suggested in one of the resolved issues in this repo, I'm using the following approach.

`function TestRive({ asset, className, animation, fit, alignment }) {
  const canvas = useRef(null);
  const animationContainer = useRef(null);
  const rive = useRef(null);

  // Start the animation
  useEffect(() => {
    rive.current = new Rive({
      src:'https://cdn.rive.app/animations/vehicles.riv',
      canvas: canvas.current,
      layout: new Layout({ fit: 'cover', alignment: 'center' }),
      autoplay: true
    });
    return () => rive.current.stop();
  }, [asset, fit, alignment]);

  return (
    <div ref={animationContainer} className={className}>
      <canvas ref={canvas} width="500" height="500" />
    </div>
  );
}`

Also tried multiple ways of loading the rive component.
`function TestRive() {
  const { rive, RiveComponent } = useRive({
    src: 'https://cdn.rive.app/animations/vehicles.riv',
    autoplay: true
  });

  if (rive) {
    console.log(rive.contents);
  }

  return (
    <div>
      <RiveComponent />
    </div>
  );
}`

These are the rules in the webpack config file.

`module.exports = {
  experiments: {
    asyncWebAssembly: true
  },
  module: {
    rules: [
      {
        test: /\.css$/,
        use: ['style-loader', 'css-loader']
      },
      {
        test: /\.scss$/,
        use: ['style-loader', 'css-loader', 'sass-loader']
      },
      {
        test: /\.(js|jsx)$/,
        use: ['babel-loader']
      },
      {
        test: /\.(ts|tsx)$/,
        exclude: /node_modules/,
        use: ['babel-loader']
      },
      {
        test: /\.(gif|jpg|png|woff|ttf|eot)$/,
        loader: 'file-loader'
      },
      {
        test: /\.svg$/,
        loader: 'svg-react-loader'
      },
      {
        test: /\.wasm$/,
        type: 'javascript/auto',
        use: ['url-loader']
      }
    ]
  }
};
`
P.S: It is working fine in any sample app created with create-react-app. But in the app where I use webpack to manually configure, it's not working as expected and throws the binding error message. What are the loaders I'm missing? Or is it something else that I'm missing?

**Comment by vigsub** [comment]
@zplata Can you help with this? I'm kinda stuck here..

**Comment by zplata** [comment]
@vigsub hello - did this help solve your issue?

**Comment by vigsub** [comment]
Oops. Sorry.. Missed this thread. Will check this and respond. Thanks :)

**Comment by vigsub** [comment]
Nope.. Not working :( .. If providing Fit.Cover and Alignment is a problem, in the second example mentioned, I don't use them. Still it's not rendering. @zplata Please assist..

**Comment by zplata** [comment]
@vigsub sorry to hear - I don't imagine it should be an issue with Webpack as you're not trying to import WASM or Rive files in your JS/TS files, but not to say that it couldn't be the issue! In case I'm missing context here, do you have a repro project that you can post here so that we can debug further and run in isolation?
